pygeoapi/models/provider/base.py

```diff
@@ -71,18 +71,33 @@ class TileMatrixSetEnumType(BaseModel):
     tileMatrixSet: str
     tileMatrixSetURI: str
     crs: str
+    tileMatrixSetDefinition: dict
 
 
 class TileMatrixSetEnum(Enum):
     WORLDCRS84QUAD = TileMatrixSetEnumType(
         tileMatrixSet="WorldCRS84Quad",
         tileMatrixSetURI="http://schemas.opengis.net/tms/1.0/json/examples/WorldCRS84Quad.json",  # noqa
-        crs="http://www.opengis.net/def/crs/OGC/1.3/CRS84"
+        crs="http://www.opengis.net/def/crs/OGC/1.3/CRS84",
+        tileMatrixSetDefinition=
+            {
+                'type': 'application/json',
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/tiling-scheme',
+                'title': 'WorldCRS84QuadTileMatrixSet definition (as JSON)',
+                'href': 'https://raw.githubusercontent.com/opengeospatial/2D-Tile-Matrix-Set/master/registry/json/WorldCRS84Quad.json'  # authoritative TMS definition
+            }
         )
     WEBMERCATORQUAD = TileMatrixSetEnumType(
         tileMatrixSet="WebMercatorQuad",
         tileMatrixSetURI="http://schemas.opengis.net/tms/1.0/json/examples/WebMercatorQuad.json",  # noqa
-        crs="http://www.opengis.net/def/crs/EPSG/0/3857"
+        crs="http://www.opengis.net/def/crs/EPSG/0/3857",
+        tileMatrixSetDefinition=
+            {
+                'type': 'application/json',
+                'rel': 'http://www.opengis.net/def/rel/ogc/1.0/tiling-scheme',
+                'title': 'WebMercatorQuadTileMatrixSet definition (as JSON)',
+                'href': 'https://raw.githubusercontent.com/opengeospatial/2D-Tile-Matrix-Set/master/registry/json/WebMercatorQuad.json'  # authoritative TMS definition
```

Comment:
@doublebyte1 Note that this link is intended to be where the Naming Authority / Definition Server "sources" its definitions, and we (the _Tiles_ SWG) hope that one day it will be possible to retrieve this from http://www.opengis.net/def/tilematrixset/OGC/1.0/WebMercatorQuad by negotiating with `Accept: application/json`.
There are various issues with the only "authoritative TMS definition" being a link to raw.githubusercontent (I think propery _Content-Type:_ was mentioned as one).

This is what issue https://github.com/opengeospatial/NamingAuthority/issues/198 is about.
Your support / input into making this happen at OGC would be most welcome :)